nodejs/examples/rabbitmq.js

```diff
@@ -0,0 +1,30 @@
+const amqp = require('amqplib/callback_api');
+const config = require("platformsh-config").config();
+
+exports.usageExample = async function() {
+    try {
+        const credentials = config.credentials('rabbitmq');
+        console.log(credentials);
+
+        let connectionString = 'amqp://' + credentials.username + ':' + credentials.password + '@' + credentials.host + ':' + credentials.port;
+        console.log(connectionString);
+
+        let connection = await amqp.connect(connectionString);
+        console.log(connection);
+
+        let channel = await connection.createChannel();
+        console.log(channel);
+
+        let queue = 'hellotest';
+        let message = 'Deploy Friday!';
+
+        await channel.assertQueue(queue, {durable: false});
+        await channel.sendToQueue(queue, Buffer.from(message))
+
+        let output = 'Message sent: ' + queue + ' ' + message;
```

Comment:
You don't mutate any variable so your `let`s can be `const`s.
This applies to all the `let` in the function.